molecule/default/verify.yml

```diff
@@ -30,12 +30,20 @@
         regexp: 'TEST_URL'
         replace: 'https://{{ ansible_ec2_public_hostname }}'
 
-    - name: Run pytest
-      delegate_to: localhost
-      vars:
-          selfsigned_cert_path: "{{ project_dir }}/configuration_files/ssl_certificates/{{ ansible_ec2_public_hostname }}.crt"
-      ansible.builtin.shell:
-        chdir: "{{ dta_dir }}"
-        cmd: |
-          export REQUESTS_CA_BUNDLE={{ selfsigned_cert_path }}
-          pytest --tb=line --configuration {{ test_config_file }} tests/ -s
+    - name: Run tests
+      block:
+        - name: Run pytest
+          delegate_to: localhost
```

Comment:
Can't we run this playbook on localhost only?

Reply:
we are addressing this in the multimachine PR

Reply:
Ah ok sorry I'm getting confused now :)